index.html

```diff
@@ -1,13 +1,85 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Tik Tak Toe</title>
-    <link rel="stylesheet" href="styles.css">
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tic Tac Toe</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="game-container">
+      <h1>Tic Tac Toe</h1>
+      <div class="player-indicators">
+        <div class="player" id="playerX">Player X</div>
+        <div class="player" id="playerO">Player O</div>
+      </div>
 
+      <div class="status">
+        <p id="turn">Player X's turn</p>
+        <p id="timer">Time left: 15s</p>
+      </div>
+      <div class="board">
+        <div class="cell" data-index="0"></div>
```

Comment:
this is redundant, avoid hardcoding also. the grid dimensions and content should be added through js.